front/src/app/pages/convention/ConventionImmersionPage.tsx

```diff
@@ -66,6 +66,18 @@ export const ConventionImmersionPage = ({
   const [displaySharedConventionMessage, setDisplaySharedConventionMessage] =
     useState(isSharedConvention);
 
+  const getPageHeaderTitle = (
+    jwt: string | undefined,
+    showSummary: boolean,
+  ) => {
+    if (jwt)
+      return showSummary
+        ? t.intro.conventionSummaryTitle
+        : t.intro.conventionEditTitle;
+
+    return t.intro.conventionTitle;
```

Comment:
est-ce que ça ne dépendait pas aussi de `showSummary` ?

```suggestion
    return showSummary ? t.intro.conventionSummaryTitle : t.intro.conventionTitle;
```

Reply:
Carrément, y'a mieux à faire du coup, je prépare ça.